CookieMediator/pom.xml

```diff
@@ -0,0 +1,102 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
if we need this mediator logic, can we include this logic into the connector. or else we need to have this separately.

Reply:
This will give two jars when build and we can use it as one ?

Reply:
Updated as requested